src/components/Calculator.js

```diff
@@ -1,40 +1,57 @@
+  }
 
-  const handleClick = (e) => {
-    setState(calculate(state, e.target.innerHTML));
-  };
+  handleClick(e) {
+    // eslint-disable-next-line react/no-access-state-in-setstate
+    this.setState(calculate(this.state, e.target.innerHTML));
+  }
 
-  return (
-    <div className="calculator">
-      <div id="calculator-display">
-        {state.total}
-        {state.operation}
-        {state.next}
+  render() {
+    const { total, next, operation } = this.state;
+    return (
+      <div className="main-container">
+        <div id="header">
+          <h1>Let&apos;s do some math!</h1>
+        </div>
+        <div className="calculator">
+          <div id="calculator-display">
+            {total}
+            {operation}
+            {next}
+          </div>
+          <button type="button" onClick={this.handleClick} className="calculator-key" id="key-clear">AC</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key" id="key-sign">+/-</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key" id="key-percent">%</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key orange-key" id="key-division">÷</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key" id="key-7">7</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key" id="key-8">8</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key" id="key-9">9</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key orange-key" id="key-multiply">x</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key" id="key-4">4</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key" id="key-5">5</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key" id="key-6">6</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key orange-key" id="key-minus">-</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key" id="key-1">1</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key" id="key-2">2</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key" id="key-3">3</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key orange-key" id="key-plus">+</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key" id="key-0">0</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key" id="key-dot">.</button>
+          <button type="button" onClick={this.handleClick} className="calculator-key orange-key" id="key-equals">=</button>
+        </div>
```

Comment:
[REQUIRED] Kindly ensure that your calculator page matches the wireframe. The text should be on the left while the calculator on the right:
![image](https://user-images.githubusercontent.com/62938735/208127977-c1382d3e-9540-4011-84d3-e70b26e2e0a0.png)

Reply:
I appreciate your work being attentive, sir.